api.go

```diff
@@ -2906,6 +2906,36 @@ func (sc *Client) doRequestDetailed(method, url, reqBody string) (*http.Response
 	return resp, err
 }
 
+// FastlyService contains details about a Fastly Service mapped a site.
+type FastlyService struct {
+	ID        string    `json:"id"`
+	AccountID string    `json:"accountID"`
+	Created   time.Time `json:"created"`
+	CreatedBy string    `json:"createdBy"`
+}
+
+// EdgeDeployment contains details about an Edge Deployment
+type EdgeDeployment struct {
+	AgentHostName    string          `json:"AgentHostName"`
+	ServicesAttached []FastlyService `json:"ServicesAttached"`
+}
+
+// ConfirmEdgeDeployment retrieves currently deployed EdgeWafs and Fastly Services mapped to a site..
```

Comment:
@jeremy-cxf should we call this `GetEdgeDeployment`?

Reply:
Makes sense, at the time was just following the docs, but yeah it doesn't make sense to call it "confirm"  :')

Reply:
Done :) Thanks!